### PR TITLE
Updating Sig Leads for federation

### PR DESCRIPTION
**Pull request by matchstick**
@csbell @quinton-hoole @sarahnovotny @thockin

### CODE REVIEW

**Comment by thockin** [comment]
If this is the SIG's will, the change LGTM.

**Comment by matchstick** [comment]
@kubernetes/sig-federation-misc We should figure out what we want to do going forward. We can discuss in the next sig meeting.

**Comment by matchstick** [comment]
I signed it!

**Comment by matchstick** [comment]
After the latest sig meeting we all agreed co-TL of quinton and csbell makes sense going forward. Updating the PR.

**Comment by matchstick** [comment]
I signed it!

**Comment by matchstick** [comment]
I have updated the "author" field to the correct email address that is validated. One more time to kick the bot.

**Comment by matchstick** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ The community meeting calendar is available as an [iCal to subscribe to] (https:
...
-| [Federation](sig-federation/README.md) | [@quinton-hoole (Quinton Hoole, Google)](https://github.com/quinton-hoole) | [Group](https://groups.google.com/forum/#!forum/kubernetes-sig-federation) | [#sig-federation](https://kubernetes.slack.com/messages/sig-federation/) | Bi-weekly on Thursdays at 9:30 AM PT on [hangouts](https://plus.google.com/hangouts/_/google.com/ubernetes) |
+| [Federation](sig-federation/README.md) | [@csbell (Christian Bell, Google)](https://github.com/csbell) <br> [@quinton-hoole (Quinton Hoole, Huawei)](https://github.com/quinton-hoole) | [Group](https://groups.google.com/forum/#!forum/kubernetes-sig-federation) | [#sig-federation](https://kubernetes.slack.com/messages/sig-federation/) | Bi-weekly on Thursdays at 9:30 AM PT on [hangouts](https://plus.google.com/hangouts/_/google.com/ubernetes) |
```
Comment (nikhiljindal) [inline README.md]: We also need to update the time from thursday to monday.

**Pull request merged**